Add log setup and configuration (#3960)
DataDog/integrations-core: ceph/README.md

dd-agent ALL=(ALL) NOPASSWD:/path/to/your/ceph
```

#### Log Collection

> 

Reviewer: ```suggestion
**Available for Agent >6.0**
```

> To enable collecting logs in the Datadog Agent, update `logs_enabled` in `datadog.yaml`:

Reviewer: ```suggestion

Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:
```

> ```

Reviewer 2: ```suggestion
```yaml
```

> ```

Reviewer 2: ```suggestion
```yaml
```

    logs_enabled: true
```

Next, edit `ceph.d/conf.yaml` by uncommenting the `logs` lines at the bottom. Update the logs `path` with the correct path to your Ceph log files.

```yaml
logs:
 - type: file
   path: /var/log/ceph/*.log
   source: ceph
   service: <APPLICATION_NAME>
```

> 

Reviewer: This should follow the template pattern found here:
https://github.com/DataDog/integrations-core/pull/3961/files

### Validation

> [Run the Agent's `status` subcommand][5] and look for `ceph` under the Checks section.

Reviewer: ```suggestion
[Run the Agent's status subcommand][5] and look for `ceph` under the Checks section.
```
